docker-compose.yml

```
@@ -5,7 +5,7 @@ services:
 
   synapse:
     build:
-        context: .github/effektio-synapse-ci
+        context: .github/effekio-synapse-ci
```

rather than doing this here, we probably want to rename the dir proper, no?
```suggestion
        context: .github/effektio-synapse-ci
```